spec.html

```diff
@@ -33170,11 +32951,9 @@ <h2>Syntax</h2>
             TemporalSign Hour HourSubcomponents[~Extended]
 
           TemporalSign :::
-            ASCIISign
-            &lt;MINUS&gt;
-
-          ASCIISign ::: one of
-            `+` `-`
+            `+`
+            `-`
```

Comment:
The Temporal proposal relies on Ecma262 defining `ASCIISign`; eliminating it seems out of scope for this PR.